pkg/util/rand/rand.go

```diff
@@ -32,6 +32,16 @@ var rng = struct {
 	rand: rand.New(rand.NewSource(time.Now().UTC().UnixNano())),
 }
 
+func Int(max int) int {
+	if max <= 0 {
```

Comment:
That one is guarding against a call to `make` and not `Intn`. `make` would also panic in this case, so that check really isn't necessary either. I would not consolidate them since they seem to be doing very different things.